docs/src/stories/customization-and-styling/OverridingStylesOfAComponentGlobally.tsx

```diff
@@ -1,7 +1,8 @@
 import { ContentOverflow, MuiThemeProvider } from "@comet/admin";
 import { createCometTheme } from "@comet/admin-theme";
 import { Typography } from "@mui/material";
-import React from "react";
+// eslint-disable-next-line @typescript-eslint/no-unused-vars, unused-imports/no-unused-imports
```

Comment:
Do we need the ESLint disables here? 🤔

Reply:
yes sadly ... the import can not be deleted with docs v2, need to update to docusaurus v3.

Reply:
But isn't the variable/import used then?

Reply:
🤔  well, ofc its used, but it think because we extend the eslint rules from `"plugin:react/jsx-runtime"` its told that he should not need it.
Should we try to turn it off again for this package, or will it be updated anyway soon?

Reply:
I tried to adapt the eslint rules to disable the new JSX Transfrom again, but was not successful :( 
Can we leave it for these few lines?

Reply:
Yes, let's leave it like this.